docs/synapse/userguides/analytical_model_tags.rstorm

```diff
@@ -36,7 +36,7 @@ This example shows the **node** for the tag ``syn:tag = rep.feye.apt1``:
 
 The ``syn:tag`` node has the following properties:
 
-- ``.created``, which is a universal property showing when the node was added to a Cortex.
+- ``.created``, which is a universal property showing when the node was added to Synapse.
```

Comment:
"added to a Cortex" seems more accurate to me than "added to Synapse" but I suppose it is probably simpler to just use Synapse to avoid confusing the target audience which probably doesn't need to know the difference.

Reply:
Kept 'Cortex' - it's a small but relevant distinction and I think this is the only place i changed it >D